railties/lib/rails/paths.rb

```diff
@@ -175,6 +175,12 @@ def to_ary
         @paths
       end
 
+      def expanded_paths
```

Comment:
@kaspth 
```
Demo::Application.config.paths["config/locales"].expanded
=> ["/app/config/locales/fr.yml", "/app/config/locales/devise.en.yml", "/app/config/locales/devise_invitable.en.yml", "/app/config/locales/en.yml", "/app/config/locales/dk.yml", "/app/config/locales/simple_form.en.yml"]
```

`expanded` returns all available files in `config/locales`, and for reloading part I just need the absolute path, which would be `/app/config/locales` in this case